pkg/terminal/terminal.go

```diff
@@ -137,6 +138,27 @@ func New(client service.Client, conf *config.Config) *Term {
 				if !firstEventBinaryInfoDownload {
 					fmt.Fprintf(t.stdout, "\n")
 				}
+			case api.EventBreakpointMaterialized:
+				bp := event.BreakpointMaterializedEventDetails.Breakpoint
+				file := bp.File
+
+				// Make the file path relative if possible.
+				if cwd, err := os.Getwd(); err == nil {
```

Comment:
You should be able to use `t.formatPath` here.

Reply:
👍 Ah, I thought there should be something like that but I looked for fields, not methods.